addons/website/static/src/snippets/s_chart/options.js

```diff
@@ -476,21 +477,22 @@ options.registry.InnerChart = options.Class.extend({
     },
     /**
      * @private
-     * @param {Event} ev
      */
-    _onMatrixInputFocusOut: function (ev) {
-        // Sometimes, an input is focusout for internal reason (like an undo
-        // recording) then focused again manually in the same JS stack
-        // execution. In that case, the blur should not trigger an option
-        // selection as the user did not leave the input. We thus defer the blur
-        // handling to then check that the target is indeed still blurred before
-        // executing the actual option selection.
-        setTimeout(() => {
```

Comment:
The removal of this needs its own commit IMO with a better explanation, like the one I added when we removed the we-input's same code. And with a reference to that commit (actually did not know there was the same code in we-matrix, otherwise I would have changed it in the same commit).